templates/profile.jsonnet

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Simulator Hi Frequency Availability - 1",
+  "name": "Simulator Testing Script - 1",
   "amountUnit": "",
   "locationId": "Matrix",
   "subscriptions": [
```

Comment:
I guess one important parameter is missing: the deviceId which points to the machine for this profile and will be its parent device